include/sparrow/layout/array_base.hpp

```diff
@@ -445,6 +463,82 @@ namespace sparrow
         return std::ranges::equal(lhs, rhs);
     }
 
-    
+    constexpr unsigned long stoul(const std::string_view str)
```

Comment:
I may have missed it, but I don't see this function used. If so, I think this should be moved in a util file instead. Also this won't work with ADL (but I guess we don't have the choice, since it is forbidden to overload in the standard namespace).

Reply:
At some point, I was working on the "parse" part to be able to change the way we format the variable. I dropped that for the moment.
Let's delete this unused function